src/features/integration/components/form/CaseForm.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+    FormControl,
+    FormGroup,
+    InputLabel,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+    TextField
+} from '@mui/material';
+import React from 'react';
+import {dropdownPlaceholder} from "../../util/DefaultValues";
+
+const CaseForm: React.FunctionComponent<any> = (props) => {
+
+    return (
+        <div>
```

Comment:
Has the div a function? You could use `<>` instead.

Reply:
removing unnecessary parent divs